src/app/api/config/config.ts

```diff
@@ -80,11 +82,19 @@ export const getAppConfig = (serverEnv: any): AppConfig => {
 		GATEWAY_API_ROOT:
 			serverEnv.NEXT_PUBLIC_GATEWAY_API_ROOT ||
 			process.env.NEXT_PUBLIC_GATEWAY_API_ROOT ||
-			'https://argo-gateway.dev.argo.cancercollaboratory.org',
+			'https://argo-gateway.argo-dev.cumulus.genomeinformatics.org',
+		GATEWAY_API_GRAPHQL:
+			serverEnv.NEXT_PUBLIC_GATEWAY_API_GRAPHQL ||
+			process.env.NEXT_PUBLIC_GATEWAY_API_GRAPHQL ||
+			'https://argo-gateway.argo-dev.cumulus.genomeinformatics.org/graphql',
 		CLINICAL_API_ROOT:
 			serverEnv.NEXT_PUBLIC_CLINICAL_API_ROOT ||
 			process.env.NEXT_PUBLIC_CLINICAL_API_ROOT ||
-			'https://argo-gateway.dev.argo.cancercollaboratory.org',
+			'https://clinical.argo-qa.cumulus.genomeinformatics.org',
+		CLINICAL_API_GRAPHQL:
+			serverEnv.NEXT_PUBLIC_CLINICAL_API_GRAPHQL ||
+			process.env.NEXT_PUBLIC_CLINICAL_API_GRAPHQL ||
+			'https://clinical.argo-qa.cumulus.genomeinformatics.org/gql',
```

Comment:
Please don't commit defaults that connect to specific environments. I would prefer a default that intentionally fails to connect. We don't want a misconfigured prod accidentally connecting to a lower env. We also don't need to be broadcasting our internal URLs.